Doc/tools/extensions/pyspecific.py

```diff
@@ -569,10 +570,13 @@ def process_audit_events(app, doctree, fromdocname):
         for i, (doc, label) in backlinks:
             if isinstance(label, str):
                 ref = nodes.reference("", nodes.Text("[{}]".format(i)), internal=True)
-                ref['refuri'] = "{}#{}".format(
-                    app.builder.get_relative_uri(fromdocname, doc),
-                    label,
-                )
+                try:
+                    ref['refuri'] = "{}#{}".format(
+                        app.builder.get_relative_uri(fromdocname, doc),
+                        label,
+                    )
+                except NoUri:
+                    pass
```

Comment:
I would make it `continue` in this case, otherwise we'll end up with `[2]` that isn't a link but also has no information about where it'd be going.

If it ends up that none of the links work (fairly likely), then it'll show up with nothing at all.

Reply:
I trust you: anyway both PDF are rendered the same: the links are out of the page :(

Reply:
No no, don't trust me on this :) I'm trusting you to get it to work